templates/perfalert_inactive_regression_needinfo.txt

```diff
@@ -0,0 +1,5 @@
+It has been {{ extra[bugid]["nweeks"] * 7 }} days with no activity on this performance regression. :{{ nickname }}, since you are the author of the regressor, bug {{ extra[bugid]["regressor_id"] }}, which triggered this performance alert, could you please provide a progress update?
```

Comment:
```suggestion
It has been over {{ extra[bugid]["nweeks"] * 7 }} days with no activity on this performance regression. :{{ nickname }}, since you are the author of the regressor, bug {{ extra[bugid]["regressor_id"] }}, which triggered this performance alert, could you please provide a progress update?
```